lib/circleci.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+'use strict';
+var http = require('https');
+
+function triggerBuild(program) {
+  return new Promise(function triggerBuildPromise(resolve, reject) {
+    var options = {
```

Comment:
@makarandp0 Nitpick:
```suggestion
    const options = {
```